### PR TITLE
Another batch of release not edits

### PR DESCRIPTION
**Pull request by jacg**
Mostly dealing with TODOs and adding missing full stops.

@wegank

### CODE REVIEW

File: nixos/doc/manual/release-notes/rl-2405.section.md [hunk 1]
```diff
@@ -568,7 +568,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   The `nimPackages` and `nim2Packages` sets have been removed.
   See https://nixos.org/manual/nixpkgs/unstable#nim for more information.
 
-- [TODO: reword to place an attribute at the front] Programs written in [D](https://dlang.org/) using the `dub` build system and package manager can now be built using `buildDubPackage` utilizing lockfiles provided by the new `dub-to-nix` helper program.
+- `buildDubPackage` can now be used to build Programs written in [D](https://dlang.org/) using the `dub` build
+  system and package manager and utilizing lockfiles provided by the new `dub-to-nix` helper program.
```
Comment (wegank) [inline nixos/doc/manual/release-notes/rl-2405.section.md:572]:
```suggestion
- `buildDubPackage` can now be used to build Programs written in [D](https://dlang.org/) using the `dub` build system and package manager.
```

I think there's no need to describe how lock files are created here, it is [in the manual](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/dlang.section.md#lockfiles-dub-lockfiles),

File: nixos/doc/manual/release-notes/rl-2405.section.md [hunk 2]
```diff
@@ -693,16 +694,16 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `youtrack` is bumped to 2023.3. The update is not performed automatically, it requires manual interaction. See the YouTrack section in the manual for details.
 
-- QtMultimedia has changed its default backend to `QT_MEDIA_BACKEND=ffmpeg` (previously `gstreamer` on Linux or `darwin` on MacOS).
+- `qtmultimedia` has changed its default backend to `QT_MEDIA_BACKEND=ffmpeg` (previously `gstreamer` on Linux or `darwin` on MacOS).
```
Comment (wegank) [inline nixos/doc/manual/release-notes/rl-2405.section.md:697]:
```suggestion
- `qt6.qtmultimedia` has changed its default backend to `QT_MEDIA_BACKEND=ffmpeg` (previously `gstreamer` on Linux or `darwin` on MacOS).
```

File: nixos/doc/manual/release-notes/rl-2405.section.md [hunk 3]
```diff
@@ -693,16 +694,16 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
...
   The previous native backends remain available but are now minimally maintained. Refer to [upstream documentation](https://doc.qt.io/qt-6/qtmultimedia-index.html#ffmpeg-as-the-default-backend) for further details about each platform.
 
 - `drbd` out-of-tree Linux kernel driver has been added in version 9.2.7. With it the DRBD 9.x features can be used instead of the 8.x features provided by the 8.4.11 in-tree driver.
 
-- [TODO: reword to place an attribute at the front] The oil shell's c++ version is now available as `oils-for-unix`. The python version is still available as `oil`
+- `oils-for-unix`, the oil shell's c++ version is now available. The python version is still available as `oil`.
```
Comment (wegank) [inline nixos/doc/manual/release-notes/rl-2405.section.md:702]:
```suggestion
- `oils-for-unix` is now the attribute name for the oil shell's c++ version. The python version is still available as `oil`.
```

File: nixos/doc/manual/release-notes/rl-2405.section.md [hunk 4]
```diff
@@ -698,7 +698,7 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `drbd` out-of-tree Linux kernel driver has been added in version 9.2.7. With it the DRBD 9.x features can be used instead of the 8.x features provided by the 8.4.11 in-tree driver.
 
-- `oils-for-unix`, the oil shell's c++ version is now available. The python version is still available as `oil`.
+- `oils-for-unix` is now the attribute name for the oil shell's c++ version. The python version is still available as `oil`.
```
Comment (wegank) [inline nixos/doc/manual/release-notes/rl-2405.section.md:701]: Sorry, I think your one is better, you can revert it if you wish...

Comment (jacg) [reply nixos/doc/manual/release-notes/rl-2405.section.md]: OK, I've removed it by force-pushing.

I was a bit confused by having an email notification about the suggestion, but not seeing it anywhere on GH: I guess you removed it before I started looking for it. (And then I re-created it myself.) Anyway, it's gone now.

**Pull request merged**